apps/print/backend/tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "noUnusedParameters": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "noUncheckedIndexedAccess": false
+    "noUncheckedIndexedAccess": false,
+    "noPropertyAccessFromIndexSignature": false
```

Comment:
My memory of this is fuzzy but I think this rule usually crops up when a type in incomplete. We can avoid blocking on this and investigate after the demo.